fastkml/kml.py

```
@@ -208,6 +210,53 @@
             element=root,
         )
 
+    def write(
```

Ruff

Missing argument descriptions in the docstring for `write`: `file_path`, `xml_declaration`

(D417)